SEM/sem_to_pif.py

```diff
@@ -0,0 +1,77 @@
+import csv
+import argparse
+from pypif import pif
+from pypif.obj import *
+
+def s3000_metadata_to_pif(closed_txt):
+
+    # create chemical system and property array
+    my_pif = ChemicalSystem()
+    my_pif.properties = []
+
+    with open(closed_txt, 'rU') as open_txt:
+        lines = open_txt.readlines()
+        for index, line in enumerate(lines):
```

Comment:
You can close the file now, so drop the following lines down one level of indentation.